pkg/azurefile/nodeserver.go

```diff
@@ -67,6 +67,7 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 			VolumeContext:     context,
 			VolumeCapability:  volCap,
 			VolumeId:          volumeID,
+			Secrets:           req.GetSecrets(),
```

Comment:
what's this change for?

Reply:
No, ephemeral volume does not need secret from node publish volume.

Reply:
Ok I'll revert this

Reply:
Done